### PR TITLE
[core] Add GraphQL CLI commands for deploy / delete actions

### PR DESCRIPTION
**Pull request by rexxars**
## Note: this is still in beta.

This PR adds commands for deploying and undeploying GraphQL APIs to the CLI.

More information can be found [here](https://www.sanity.io/help/graphql-beta)

Moving this into next in order to make it easier to try out for our users, as well as keeping it in sync with the other improvements over time.

### CODE REVIEW

File: packages/@sanity/cli/src/util/noSuchCommandText.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const coreCommands = [
   'build',
   'check',
   'configcheck',
+  'cors',
```
Comment (bjoerge) [inline packages/@sanity/cli/src/util/noSuchCommandText.js]: Is this intentional? Doesn't seem to be related to cors?

File: packages/@sanity/core/src/actions/graphql/deleteApiAction.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+module.exports = async function deployApiActions(args, context) {
```
Comment (bjoerge) [inline packages/@sanity/core/src/actions/graphql/deleteApiAction.js]: Should the function be named `deleteApiAction` here?

File: packages/@sanity/core/src/commands/graphql/deleteGraphQLAPICommand.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const lazyRequire = require('@sanity/util/lib/lazyRequire')
+
+export default {
+  name: 'delete',
```
Comment (bjoerge) [inline packages/@sanity/core/src/commands/graphql/deleteGraphQLAPICommand.js]: Did we consider `sanity graphql undeploy`? Since we went for `sanity undeploy` for content studio, I feel it's a bit more natural to name it undeploy here as well.

File: packages/@sanity/core/src/actions/graphql/extractFromSanitySchema.js [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+        fieldName: '_rev',
+        type: 'String',
+        isNullable: false,
+        description: 'Current document revision'
+      }
+    ]
+  }
+
+  function arrayify(thing) {
+    return Array.isArray(thing) ? thing : [thing]
```
Comment (bjoerge) [inline packages/@sanity/core/src/actions/graphql/extractFromSanitySchema.js]: Depending on how `arrayify` is called, this could potentially yield `[undefined]` if `thing` is `undefined`. Maybe do an additional check and return an empty array in that case.

File: packages/@sanity/cli/src/util/noSuchCommandText.js (hunk 1 above)
Comment (rexxars) [reply packages/@sanity/cli/src/util/noSuchCommandText.js]: I realized it was missing from the array while editing the file - didn't think to open a separate PR for it. 🤷‍♂️

File: packages/@sanity/core/src/actions/graphql/deleteApiAction.js (hunk 2 above)
Comment (rexxars) [reply packages/@sanity/core/src/actions/graphql/deleteApiAction.js]: Whoops, thanks!

File: packages/@sanity/core/src/commands/graphql/deleteGraphQLAPICommand.js (hunk 3 above)
Comment (rexxars) [reply packages/@sanity/core/src/commands/graphql/deleteGraphQLAPICommand.js]: True.

File: packages/@sanity/cli/src/util/noSuchCommandText.js (hunk 1 above)
Comment (bjoerge) [reply packages/@sanity/cli/src/util/noSuchCommandText.js]: Ah, no need for a separate PR I think - but should'nt `graphql` be in that list too?

Comment (rexxars) [reply packages/@sanity/cli/src/util/noSuchCommandText.js]: Not yet, we don't want to explicitly hint at it before it's "public"
